packages/components/src/components/input/input.scss

```diff
@@ -132,7 +132,7 @@
     &__label {
         color: var(--text-less-prominent);
         font-size: var(--text-size-xs);
-        background-color: var(--fill-normal);
+        background-color: var(--general-main-2);
```

Comment:
@aliafsah-fs you cannot change the background color here, the correct solution is not to set a specific bg and inherit it from it container.
there is two cases 
1- label is in the middle of input which can have transparent BG
2- label is on left top of input which it should overlay the top bored with the BG of its parent

Reply:
It's not related to this PR, lets discuss it on another one. I'm reverting it.